src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -318,7 +318,11 @@ private void findUnitsThatCantMove(final Map<Territory, ProPurchaseTerritory> pu
           if (ta != null) {
             production = ta.getProduction();
           }
-          if (territoriesToDefendWithOneUnit.contains(t) && unitValue <= (production + 3)) {
+
+          // Only defend territories that either already have units (avoid abandoning territories)
+          // or where unit value is less than production + 3 (avoid sacrificing expensive units to block)
+          if (territoriesToDefendWithOneUnit.contains(t)
```

Comment:
parens really helped  :+1: 

The logic IMO is still a bit difficult to follow. No strong feelings on how to improve it one way or another particularly.